src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala

```diff
@@ -228,6 +229,11 @@ private[redshift] object Parameters {
      */
     def useStagingTable: Boolean = parameters("usestagingtable").toBoolean
 
+    /**
+     * Enable case sensitiveness of column identifiers.
+     */
```

Comment:
iirc there is something which needs to be set on the redshift cluster to make case sensitiveness valid?  Can you mention what that is in the docstring here?